worker/pkg/workflows/datasync/activities/gen-benthos-configs/processors_test.go

```diff
@@ -335,71 +343,9 @@ func Test_buildProcessorConfigsJavascript_DeepKeys(t *testing.T) {
 let programOutput = undefined;
 const benthos = {
   v0_msg_as_structured: () => ({foo: {bar: {baz: "world"}}}),
-  v0_msg_set_structured: (val) => {
```

Comment:
Hard to tell based on the diff, but did these two tests just get combined?

Reply:
yeah it became a duplicate test since I moved the setNestedProperty logic out of the javascript